### ISSUE TITLE
Cloud build for iOS fails with 'operation not permitted, mkdir'

### ISSUE THREAD

**Issue by ntrpnr** [NativeScript/sidekick-feedback#226]
### Did you verify whether this issue has already been reported here?
Yes

### Tell us about the problem
My iOS cloud build fails with the message ```EPERM: operation not permitted, mkdir '/tmp/runtimeDir11869-39506-xdqhqg.kxa0ad1jor'```

### Which platform(s) does the issue occur on?
iOS (Cloud build) - local build works fine.
macOS

### Provide the following version numbers that the issue occurs with:

- **NativeScript Sidekick Desktop:** 1.10.3-v.2018.6.26.1 (latest)
**NativeScript CLI version:** 4.1.0
**CLI extension nativescript-cloud version:** 1.12.0
**CLI extension nativescript-starter-kits version:** 0.3.4

### Tell us how to recreate the issue in as much detail as possible
1. Start a cloud build for iOS

### Does this issue happen every time?
Yes, I've tried to do a clean build as well.

### Send the client logs
https://gist.github.com/ntrpnr/e65f92e0a6258f7a0c7bfe27d76cd754

**Comment by rosen-vladimirov** [comment]
Hey @ntrpnr ,
Thanks for reporting this issue. It looks like you are using a `next` version of `tns-ios`, so when trying to build your app in the cloud, we try to use the `next` version of NativeScript CLI. Unfortunately, one of my latest commits in NativeScript CLI causes issues with the sandboxed environment in the cloud. We'll fix these issues soon, but meanwhile you have two ways to continue working:
1. Downgrade your `tns-ios` version (you can execute `tns platform remove ios && tns platform add ios` in your terminal.
2. Set the version of NativeScript CLI that will be used in the cloud to be `4.1.2`. In order to do this, you can create a file `~/.local/share/.nativescript-cli/cloud-configuration.json` with the following content:
```JSON
{
	"tnsCliCloudVersion": "4.1.2"
}
```

Can you give it a try and see if it works for you?

**Comment by ntrpnr** [comment]
So I went with option 2:
Created the file and pasted the content. Restarted Sidekick.
Started a cloud build.

I did get another error this time:
```** ARCHIVE FAILED **````
This is the same error as #225, but the local build works great for me.
If the logging in cloud build builds were more verbose I would be able to troubleshoot this better myself, I think.


Log:
https://gist.github.com/ntrpnr/885dfd3f4299ead1cc89cfa91b82e682
